pkg/proxy/apis/config/types.go

```diff
@@ -98,6 +98,12 @@ type KubeProxyNFTablesConfiguration struct {
 	// '1m', '2h22m'). A value of 0 means every Service or EndpointSlice change will
 	// result in an immediate iptables resync.
 	MinSyncPeriod metav1.Duration
+	// allowOlderKernel causes the minimum-kernel-version check to be bypassed. This
+	// will allow running nftables kube-proxy on nodes running older distros, but it
+	// should only be used if you know nothing else on the system is using nftables,
+	// since nft 0.9.8 and older will crash if kube-proxy's nftables rules are
+	// present.
+	AllowOlderKernel bool
```

Comment:
Should this be more specific?

AllowNFTablesLEQ_0_9_8

I have some concerns that `AllowOlderKernel` is pretty vague as a flag name.

Reply:
The vagueness is intentional, since we can't actually check for the thing that we want to check for (host filesystem's `nft` is 0.9.8 or later), so we're doing something vague to approximate it.

This flag will hopefully go away in GA anyway because (a) by that point the "older kernels" will be old enough that we don't have to feel as guilty about not supporting them, and (b) the problems caused by too-old-`nft` are annoying enough that we don't really want anybody using this option in production.

Probably I should document that better...

Reply:
OK, renamed this to `skipKernelVersionCheck` and tried to clarify the docs. I didn't say anything about removing this in GA because maybe we won't.